src/Core/Configure/Engine/DbConfig.php

```diff
@@ -62,11 +62,15 @@ public function __construct($table = null, $cacheConfig = 'default')
      */
     public function read($key)
     {
-        return $this->_table
-            ->find('kv')
-            ->where([
-                $this->_table->aliasField('namespace') => $key
-            ])
+        $query = $this->_table->find('kv');
+
+        if ($key !== '*') {
+            $query->where([
+                $this->_table->aliasField('namespace') . ' IS' => $key
```

Comment:
So the ORM can work with `null` or any other value.
http://book.cakephp.org/3.0/en/orm/query-builder.html#automatic-is-null-creation